.github/workflows/api-schema-lint.yml

```diff
@@ -43,4 +43,5 @@ jobs:
           python3 folio-tools/api-schema-lint/api_schema_lint.py \
             --loglevel info \
             --directories ${{ env.API_DIRECTORIES }} \
-            --excludes ${{ env.API_EXCLUDES }}
+            --excludes ${{ env.API_EXCLUDES }} \
+            -w
```

Comment:
As per the instructions at the top of the api-schema-lint.yml GitHub Workflow file, this one does not have any warnings parameter.